src/partition/repart.c

```diff
@@ -173,7 +175,9 @@ STATIC_DESTRUCTOR_REGISTER(arg_image, freep);
 STATIC_DESTRUCTOR_REGISTER(arg_definitions, strv_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_key, erase_and_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_private_key, EVP_PKEY_freep);
+STATIC_DESTRUCTOR_REGISTER(arg_private_key_path, freep);
```

Comment:
this isn't necessarily a path, is it? it can be basically anything, can't it? hence maybe name this `arg_private_key_provider_name` or so?

Reply:
or `arg_private_key_uri`?